requirements.txt

```diff
@@ -1,2 +1,2 @@
-jira==3.2.0
+jira @ git+https://github.com/jpavlav/jira.git@40cc086c2dee4f57b09b589e73eeea72721c9802
```

Comment:
We should not make this pack dependent on your fork of `pycontribs/jira`. It looks like you opened https://github.com/pycontribs/jira/pull/1837 pending review by maintainers.
I feel we should wait until this is merged upstream and just bump the version of `jira` once those changes are accepted before proceeding with this PR. Otherwise we risk merging changes that depend on something that has diverged from the well-supported upstream `jira` python library in favor of your fork.

Reply:
Great! I think so too. I've been chatting with the maintainers over there and I think someone will get around to taking a look in a few days. Either way, thanks for looking this over.

Reply:
Okay, we should be good here. The changes that I made in https://github.com/pycontribs/jira/pull/1837 were merged and included in [this release](https://github.com/pycontribs/jira/releases/tag/3.7.0). The maintainers over there included some additional code changes in a subsequent release and published `3.8.0` to `pypi`.